fractal_server/app/routes/api/v1/workflow.py

```diff
@@ -58,6 +58,9 @@ async def get_workflow_list(
     project = await _get_project_check_owner(
         project_id=project_id, user_id=user.id, db=db
     )
+    for wf in project.workflow_list:
```

Comment:
Could we avoid the `for` loop?

Reply:
Fixed, as of https://github.com/fractal-analytics-platform/fractal-server/pull/1082/commits/f31b72c6cd95a2d072a7dba9851da4c451e70bbb.